packages/dds/tree/src/simple-tree/treeNodeApi.ts

```diff
@@ -117,6 +118,8 @@ export interface TreeNodeApi {
 	 *
 	 * If the node has more than one identifier, then this will throw an error.
 	 *
+	 * If the node is unhydrated and using the field default, this will throw an error.
```

Comment:
This doc already refers to field-default-generated identifiers as "automatically generated" identifiers. Let's use the same terminology so as not to confuse the user into thinking they might be two different concepts. Same for "unhydrated" - many users won't know what that term means. And, we should probably roll this into the first case in the doc, since that's the case that is talking about UUIDs.

"If the node's identifier is a valid UUID... to that identifier. Note that the node must already have been inserted into the tree in order to retrieve a generated UUID (or `shortId` will error). This is useful..."

Reply:
Updated doc with suggested change